RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -526,7 +526,7 @@
     PFTauProducer = cms.InputTag('hpsPFTauProducer'),
     Prediscriminants = requireDecayMode.clone(),
     loadMVAfromDB = cms.bool(True),
-    mvaName = cms.string("RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v1"),
+    mvaName = cms.string("RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v2"),
```

Comment:
what is the rationale to keep the same name of the producer with "v1"?
Should it be changed as well to v2?

Reply:
I had the impression that this version number is not really used because the year does not even show up, so v1 would not stand for 2017v1. But the naming history goes beyond my scope. @steggema @rmanzoni @mbluj could you comment?

Reply:
@mbluj may know better, but we have two things to consider,

a) the list of input variables, which I think can be considered to be represented by the tag "MVArun2v1DBoldDMwLT" and then resolved to "mvaOpt = cms.string("DBoldDMwLTwGJ")"
b) the version of the training, which is 2017v2

So I think it may be fine to keep the version as Sebastian has it. OTOH, maybe a case can be made for the training tag (2017v2) representing everything, but that may need to lead to even larger overall changes...

Reply:
I confirm what @steggema wrote. Name of module in the python configuration represents version of MVA, i.e. list of inputs, while name of payloads stands for a given training of the MVA. 
From practical perspective, it is also useful to keep names of modules and change only names of payloads as it reduces number of modifications (in terms of no. of modified lines and files).

Reply:
Looking back to CMSSW_7_6_2, when this naming pattern was introduced, the previous payloads had the version of the MVA in the name (or at least it seemed like it).
Please share a pointer to a past example of a different case of ID where the payload does not contain the "vX" of the MVA in its name, just to be clear of a somewhat established practice.

Reply:
> In this PR #21022 the 2017v1

yet, it was still a "v1".
I just wanted to understand if this "vX" is becoming more formalized just now.
Perhaps for consistency of future changes you can make a note in some tau POG area to have that as a reference.

Reply:
I see what you mean I agree that some version bookkeeping will be useful for final users who usually know name of a producer or/and access name within PAT (and a release version), but not a name of a related payload. It is up to conveners to decide if it makes sense to change name of producer or it will be enough to provide a table in Tau POG recommendation TWiki.

Reply:
OK, I can wait until Thursday to signoff on this PR (considering the holiday tomorrow).
I hope that by then the decision to keep the current naming or to change it can be made.

Reply:
@steggema and I agreed to keep the current naming.

The reason is not to force analysts to adjust their code now while we expect to move to recommend a newer DNN-based Tau ID sometime soon, which will demand code changes. 

With that, we’ll have a chance to start right away with a more coherent version naming.

Reply:
OK, I will sign off, but some better formalized naming would be quite helpful.
Without being aware of this discussion and digging into the git history, it would appear more and more that the configuration may be buggy.